File: backend/src/main/java/pl/gddkia/estimate/AddNewEstimateRest.java
package pl.gddkia.estimate;


public record AddNewEstimateRest(
    String companyName,
    String contractName,
    String dateFrom,
    String dateTo,
    String branchName,
    String[] regionName,
    String[] sectionName,
    Long roadLength
) {

}

File: backend/src/main/java/pl/gddkia/estimate/AddNewEstimateRest.kt
package pl.gddkia.estimate


data class AddNewEstimateRest(
    val companyName: String,
    val contractName: String,
    val dateFrom: String?,
    val dateTo: String?,
    val branchName: String,
    val regionName: String,
    val sectionName: String,
    val roadLength: Long
)
